baselines/TrailingCommas.hx.baseline.txt

```diff
@@ -0,0 +1,822 @@
+            ^
+            source.hx meta.method.hx meta.method.block.hx
+             ^^^^^^^^
+             source.hx meta.method.hx meta.method.block.hx entity.name.function.hx
+                     ^
+                     source.hx meta.method.hx meta.method.block.hx meta.brace.round.hx
+                      ^
+                      source.hx meta.method.hx meta.method.block.hx meta.brace.round.hx
+                       ^
+                       source.hx meta.method.hx meta.method.block.hx punctuation.terminator.hx
+>}
+ ^
+ source.hx meta.method.hx meta.method.block.hx punctuation.definition.block.end.hx
+>
+ ^
+ source.hx
+>function funlong(
+ ^^^^^^^^
+ source.hx meta.method.hx storage.type.function.hx
+         ^
+         source.hx meta.method.hx
+          ^^^^^^^
+          source.hx meta.method.hx entity.name.function.hx
+                 ^
+                 source.hx meta.method.hx meta.parameters.hx punctuation.definition.parameters.begin.hx
+>	a:Int, // a
+ ^
+ source.hx meta.method.hx meta.parameters.hx
+  ^
+  source.hx meta.method.hx meta.parameters.hx variable.parameter.hx
+   ^
+   source.hx meta.method.hx meta.parameters.hx keyword.operator.type.annotation.hx
+    ^^^
+    source.hx meta.method.hx meta.parameters.hx support.class.builtin.hx
+       ^
+       source.hx meta.method.hx meta.parameters.hx punctuation.separator.comma.hx
+        ^
+        source.hx meta.method.hx meta.parameters.hx
+         ^^
+         source.hx meta.method.hx meta.parameters.hx comment.line.double-slash.hx punctuation.definition.comment.hx
+           ^^^
+           source.hx meta.method.hx meta.parameters.hx comment.line.double-slash.hx
+>	callback:(?a:(f:Int,)->Int, ?b:Int)->Int, // cb
+ ^
+ source.hx meta.method.hx meta.parameters.hx
+  ^^^^^^^^
+  source.hx meta.method.hx meta.parameters.hx variable.parameter.hx
+          ^
+          source.hx meta.method.hx meta.parameters.hx keyword.operator.type.annotation.hx
+           ^
+           source.hx meta.method.hx meta.parameters.hx punctuation.definition.parameters.begin.hx
+            ^
+            source.hx meta.method.hx meta.parameters.hx keyword.operator.optional.hx
+             ^
+             source.hx meta.method.hx meta.parameters.hx variable.parameter.hx
+              ^
+              source.hx meta.method.hx meta.parameters.hx keyword.operator.type.annotation.hx
+               ^
+               source.hx meta.method.hx meta.parameters.hx punctuation.definition.parameters.begin.hx
+                ^
+                source.hx meta.method.hx meta.parameters.hx variable.parameter.hx
+                 ^
+                 source.hx meta.method.hx meta.parameters.hx keyword.operator.type.annotation.hx
+                  ^^^
+                  source.hx meta.method.hx meta.parameters.hx support.class.builtin.hx
+                     ^
+                     source.hx meta.method.hx meta.parameters.hx
```

Comment:
should be a "comma", or is that what the other PR fixes?

there are some more places where it fails to detect a comma, it'll be easier to spot once the source is converted from tabs.

Reply:
Yep, there is still a lot of skipped commas it seems, but it was like this before too? At least coloring is much better.

There is still one broken case in test:
```haxe
var a = (
    a:Int,
    b:Int, // bad b color, () is detected as block instead of arrow
) -> 0;
```
Do i need to remove it and move to separate issue for now?

Reply:
Not this time, just tried it a bit and still cannot understand why multiline arrows is not handled

Reply:
then we'll have to keep #57 open for now.